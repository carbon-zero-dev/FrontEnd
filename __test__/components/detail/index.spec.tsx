/**
 * @jest-environment jsdom
 */

import React, { Suspense } from 'react';
import {
	certified_product,
	non_certified_product,
} from '../../../utils/dummtData';
import { render, screen } from '@testing-library/react';

import Detail from '../../../components/Detail';
import { RecoilRoot } from 'recoil';

describe('DetailPage 스냅샷 테스트', () => {
	test('친환경 인증 제품', () => {
		const el = render(
			<RecoilRoot>
				<Suspense fallback={<div>Loading... </div>}>
					<Detail testProduct={certified_product} />
				</Suspense>
			</RecoilRoot>,
		);
		expect(el).toMatchSnapshot();
	});
	test('일반 제품', () => {
		const el = render(
			<RecoilRoot>
				<Suspense fallback={<div>Loading... </div>}>
					<Detail testProduct={non_certified_product} />
				</Suspense>
			</RecoilRoot>,
		);
		expect(el).toMatchSnapshot();
	});
});
