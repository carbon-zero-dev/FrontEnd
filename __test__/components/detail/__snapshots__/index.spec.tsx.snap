// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailPage 스냅샷 테스트 일반 제품 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="layout_container"
      >
        <div
          class="sc-crzoAE hXnyKs layout_container"
        >
          <img
            alt="상품 이미지"
            src="https://cdn.pixabay.com/photo/2020/07/21/16/24/landscape-5426755_640.jpg"
          />
          <div
            class="sc-jSFjdj iUSrcn"
          >
            <div
              class="sc-gKAaRy gQlVXN"
            >
              <h2
                class="title"
              >
                Dummy Name
              </h2>
              <h3
                class="price"
              >
                1000
                 원
              </h3>
            </div>
            <div
              class="sc-iCoGMd cAGgJW"
            >
              <p>
                <span>
                  삼성카드 간편결제 시 5% 청구할인
                </span>
                 (최대 2천원)
              </p>
            </div>
            <div
              class="sc-fujyAs htYleG"
            >
              <p
                class="title"
              >
                탄소 노노 유저만을 위한 혜택
              </p>
              <p
                class="max-point"
              >
                최대 적립 포인트: 
                10
                원
              </p>
              <p
                class="basic-point"
              >
                 ㄴ기본적립: 
                4
                원
              </p>
              <p
                class="extra-point"
              >
                맴버십 추가 적립: 
                6
                원
              </p>
            </div>
            <div
              class="sc-pNWdM ghyfJQ"
            >
              <p
                class="title"
              >
                추가상품
              </p>
              <div
                class="MuiFormControl-root makeStyles-formControl-7"
              >
                <label
                  class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
                  data-shrink="false"
                  id="demo-simple-select-outlined-label"
                >
                  추가상품 1
                </label>
                <div
                  class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
                >
                  <div
                    aria-haspopup="listbox"
                    aria-labelledby="demo-simple-select-outlined-label demo-simple-select-outlined"
                    class="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input"
                    id="demo-simple-select-outlined"
                    role="button"
                    tabindex="0"
                  >
                    <span>
                      ​
                    </span>
                  </div>
                  <input
                    aria-hidden="true"
                    class="MuiSelect-nativeInput"
                    tabindex="-1"
                    value=""
                  />
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M7 10l5 5 5-5z"
                    />
                  </svg>
                  <fieldset
                    aria-hidden="true"
                    class="PrivateNotchedOutline-root-9 MuiOutlinedInput-notchedOutline"
                  >
                    <legend
                      class="PrivateNotchedOutline-legendLabelled-11"
                    >
                      <span>
                        additional-product
                      </span>
                    </legend>
                  </fieldset>
                </div>
              </div>
            </div>
            <div
              class="sc-kEqXSa lnzvgU"
            >
              <div
                class="sc-jrsJWt isZYCS"
              >
                <button
                  aria-label="delete"
                  class="MuiButtonBase-root MuiIconButton-root"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiIconButton-label"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 13H5v-2h14v2z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
                <h4
                  class="quantity"
                >
                  1
                </h4>
                <button
                  aria-label="delete"
                  class="MuiButtonBase-root MuiIconButton-root"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiIconButton-label"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </div>
              <span
                class="total_quantity"
              >
                총 수량:  1개  |  
              </span>
              <h3
                class="total_price"
              >
                총 가격: 1000원
              </h3>
            </div>
            <div
              class="sc-iqAclL diZOqr"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                focusable="false"
                style="width: 100px; height: 100px; color: rgb(229, 229, 229);"
                viewBox="0 0 24 24"
              >
                <path
                  d="M15.55 13c.75 0 1.41-.41 1.75-1.03l3.58-6.49c.37-.66-.11-1.48-.87-1.48H5.21l-.94-2H1v2h2l3.6 7.59-1.35 2.44C4.52 15.37 5.48 17 7 17h12v-2H7l1.1-2h7.45zM6.16 6h12.15l-2.76 5H8.53L6.16 6zM7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zm10 0c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2z"
                />
              </svg>
              <p>
                이 상품은 현재 구매하실 수 없는 상품입니다.
              </p>
              <p>
                (재입고 시 구매가능)
              </p>
            </div>
          </div>
        </div>
        <div
          class="sc-bqGGPW fiXlQZ"
        >
          <h2>
            이러한 제품들을 사용해 보는 건 어떠세요?
          </h2>
          <div
            class="sc-ksluID kCniuP"
          >
            <a
              class="sc-hBMUJo hsXmSB"
              href="/detail/1"
            >
              <div
                class="sc-dIsUp fYttkb"
              >
                <img
                  alt="추천 상품 이미지"
                />
                <p>
                  Dummy Re...
                </p>
                <p>
                  2000
                   원
                </p>
              </div>
            </a>
          </div>
        </div>
        <div
          class="sc-bdnxRM deIdvJ"
        >
          <div
            class="sc-bdnxRM deIdvJ"
          />
        </div>
        <div
          class="sc-eCApnc jPPoLs"
        >
          <p>
            Dummy Description
          </p>
          <img
            alt="제품 상세 이미지"
            src="https://cdn.pixabay.com/photo/2020/07/21/16/24/landscape-5426755_640.jpg"
          />
          <p>
            Dummy Description
          </p>
          <img
            alt="제품 상세 이미지"
            src="https://cdn.pixabay.com/photo/2020/07/21/16/24/landscape-5426755_640.jpg"
          />
          <p>
            Dummy Description
          </p>
          <img
            alt="제품 상세 이미지"
            src="https://cdn.pixabay.com/photo/2020/07/21/16/24/landscape-5426755_640.jpg"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="layout_container"
    >
      <div
        class="sc-crzoAE hXnyKs layout_container"
      >
        <img
          alt="상품 이미지"
          src="https://cdn.pixabay.com/photo/2020/07/21/16/24/landscape-5426755_640.jpg"
        />
        <div
          class="sc-jSFjdj iUSrcn"
        >
          <div
            class="sc-gKAaRy gQlVXN"
          >
            <h2
              class="title"
            >
              Dummy Name
            </h2>
            <h3
              class="price"
            >
              1000
               원
            </h3>
          </div>
          <div
            class="sc-iCoGMd cAGgJW"
          >
            <p>
              <span>
                삼성카드 간편결제 시 5% 청구할인
              </span>
               (최대 2천원)
            </p>
          </div>
          <div
            class="sc-fujyAs htYleG"
          >
            <p
              class="title"
            >
              탄소 노노 유저만을 위한 혜택
            </p>
            <p
              class="max-point"
            >
              최대 적립 포인트: 
              10
              원
            </p>
            <p
              class="basic-point"
            >
               ㄴ기본적립: 
              4
              원
            </p>
            <p
              class="extra-point"
            >
              맴버십 추가 적립: 
              6
              원
            </p>
          </div>
          <div
            class="sc-pNWdM ghyfJQ"
          >
            <p
              class="title"
            >
              추가상품
            </p>
            <div
              class="MuiFormControl-root makeStyles-formControl-7"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
                data-shrink="false"
                id="demo-simple-select-outlined-label"
              >
                추가상품 1
              </label>
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
              >
                <div
                  aria-haspopup="listbox"
                  aria-labelledby="demo-simple-select-outlined-label demo-simple-select-outlined"
                  class="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input"
                  id="demo-simple-select-outlined"
                  role="button"
                  tabindex="0"
                >
                  <span>
                    ​
                  </span>
                </div>
                <input
                  aria-hidden="true"
                  class="MuiSelect-nativeInput"
                  tabindex="-1"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7 10l5 5 5-5z"
                  />
                </svg>
                <fieldset
                  aria-hidden="true"
                  class="PrivateNotchedOutline-root-9 MuiOutlinedInput-notchedOutline"
                >
                  <legend
                    class="PrivateNotchedOutline-legendLabelled-11"
                  >
                    <span>
                      additional-product
                    </span>
                  </legend>
                </fieldset>
              </div>
            </div>
          </div>
          <div
            class="sc-kEqXSa lnzvgU"
          >
            <div
              class="sc-jrsJWt isZYCS"
            >
              <button
                aria-label="delete"
                class="MuiButtonBase-root MuiIconButton-root"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiIconButton-label"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 13H5v-2h14v2z"
                    />
                  </svg>
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <h4
                class="quantity"
              >
                1
              </h4>
              <button
                aria-label="delete"
                class="MuiButtonBase-root MuiIconButton-root"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiIconButton-label"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                    />
                  </svg>
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
            </div>
            <span
              class="total_quantity"
            >
              총 수량:  1개  |  
            </span>
            <h3
              class="total_price"
            >
              총 가격: 1000원
            </h3>
          </div>
          <div
            class="sc-iqAclL diZOqr"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
              focusable="false"
              style="width: 100px; height: 100px; color: rgb(229, 229, 229);"
              viewBox="0 0 24 24"
            >
              <path
                d="M15.55 13c.75 0 1.41-.41 1.75-1.03l3.58-6.49c.37-.66-.11-1.48-.87-1.48H5.21l-.94-2H1v2h2l3.6 7.59-1.35 2.44C4.52 15.37 5.48 17 7 17h12v-2H7l1.1-2h7.45zM6.16 6h12.15l-2.76 5H8.53L6.16 6zM7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zm10 0c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2z"
              />
            </svg>
            <p>
              이 상품은 현재 구매하실 수 없는 상품입니다.
            </p>
            <p>
              (재입고 시 구매가능)
            </p>
          </div>
        </div>
      </div>
      <div
        class="sc-bqGGPW fiXlQZ"
      >
        <h2>
          이러한 제품들을 사용해 보는 건 어떠세요?
        </h2>
        <div
          class="sc-ksluID kCniuP"
        >
          <a
            class="sc-hBMUJo hsXmSB"
            href="/detail/1"
          >
            <div
              class="sc-dIsUp fYttkb"
            >
              <img
                alt="추천 상품 이미지"
              />
              <p>
                Dummy Re...
              </p>
              <p>
                2000
                 원
              </p>
            </div>
          </a>
        </div>
      </div>
      <div
        class="sc-bdnxRM deIdvJ"
      >
        <div
          class="sc-bdnxRM deIdvJ"
        />
      </div>
      <div
        class="sc-eCApnc jPPoLs"
      >
        <p>
          Dummy Description
        </p>
        <img
          alt="제품 상세 이미지"
          src="https://cdn.pixabay.com/photo/2020/07/21/16/24/landscape-5426755_640.jpg"
        />
        <p>
          Dummy Description
        </p>
        <img
          alt="제품 상세 이미지"
          src="https://cdn.pixabay.com/photo/2020/07/21/16/24/landscape-5426755_640.jpg"
        />
        <p>
          Dummy Description
        </p>
        <img
          alt="제품 상세 이미지"
          src="https://cdn.pixabay.com/photo/2020/07/21/16/24/landscape-5426755_640.jpg"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`DetailPage 스냅샷 테스트 친환경 인증 제품 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="layout_container"
      >
        <div
          class="sc-crzoAE hXnyKs layout_container"
        >
          <img
            alt="상품 이미지"
            src="https://cdn.pixabay.com/photo/2020/07/21/16/24/landscape-5426755_640.jpg"
          />
          <div
            class="sc-jSFjdj iUSrcn"
          >
            <div
              class="sc-gKAaRy gQlVXN"
            >
              <h2
                class="title"
              >
                Dummy Name
              </h2>
              <h3
                class="price"
              >
                1000
                 원
              </h3>
            </div>
            <div
              class="sc-iCoGMd cAGgJW"
            >
              <p>
                <span>
                  삼성카드 간편결제 시 5% 청구할인
                </span>
                 (최대 2천원)
              </p>
            </div>
            <div
              class="sc-fujyAs htYleG"
            >
              <p
                class="title"
              >
                탄소 노노 유저만을 위한 혜택
              </p>
              <p
                class="max-point"
              >
                최대 적립 포인트: 
                10
                원
              </p>
              <p
                class="basic-point"
              >
                 ㄴ기본적립: 
                4
                원
              </p>
              <p
                class="extra-point"
              >
                맴버십 추가 적립: 
                6
                원
              </p>
            </div>
            <div
              class="sc-pNWdM ghyfJQ"
            >
              <p
                class="title"
              >
                추가상품
              </p>
              <div
                class="MuiFormControl-root makeStyles-formControl-1"
              >
                <label
                  class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
                  data-shrink="false"
                  id="demo-simple-select-outlined-label"
                >
                  추가상품 1
                </label>
                <div
                  class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
                >
                  <div
                    aria-haspopup="listbox"
                    aria-labelledby="demo-simple-select-outlined-label demo-simple-select-outlined"
                    class="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input"
                    id="demo-simple-select-outlined"
                    role="button"
                    tabindex="0"
                  >
                    <span>
                      ​
                    </span>
                  </div>
                  <input
                    aria-hidden="true"
                    class="MuiSelect-nativeInput"
                    tabindex="-1"
                    value=""
                  />
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M7 10l5 5 5-5z"
                    />
                  </svg>
                  <fieldset
                    aria-hidden="true"
                    class="PrivateNotchedOutline-root-3 MuiOutlinedInput-notchedOutline"
                  >
                    <legend
                      class="PrivateNotchedOutline-legendLabelled-5"
                    >
                      <span>
                        additional-product
                      </span>
                    </legend>
                  </fieldset>
                </div>
              </div>
            </div>
            <div
              class="sc-kEqXSa lnzvgU"
            >
              <div
                class="sc-jrsJWt isZYCS"
              >
                <button
                  aria-label="delete"
                  class="MuiButtonBase-root MuiIconButton-root"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiIconButton-label"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 13H5v-2h14v2z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
                <h4
                  class="quantity"
                >
                  1
                </h4>
                <button
                  aria-label="delete"
                  class="MuiButtonBase-root MuiIconButton-root"
                  tabindex="0"
                  type="button"
                >
                  <span
                    class="MuiIconButton-label"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                      />
                    </svg>
                  </span>
                  <span
                    class="MuiTouchRipple-root"
                  />
                </button>
              </div>
              <span
                class="total_quantity"
              >
                총 수량:  1개  |  
              </span>
              <h3
                class="total_price"
              >
                총 가격: 1000원
              </h3>
            </div>
            <div
              class="sc-iqAclL diZOqr"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
                focusable="false"
                style="width: 100px; height: 100px; color: rgb(229, 229, 229);"
                viewBox="0 0 24 24"
              >
                <path
                  d="M15.55 13c.75 0 1.41-.41 1.75-1.03l3.58-6.49c.37-.66-.11-1.48-.87-1.48H5.21l-.94-2H1v2h2l3.6 7.59-1.35 2.44C4.52 15.37 5.48 17 7 17h12v-2H7l1.1-2h7.45zM6.16 6h12.15l-2.76 5H8.53L6.16 6zM7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zm10 0c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2z"
                />
              </svg>
              <p>
                이 상품은 현재 구매하실 수 없는 상품입니다.
              </p>
              <p>
                (재입고 시 구매가능)
              </p>
            </div>
          </div>
        </div>
        <div
          class="sc-bqGGPW fiXlQZ"
        >
          <h2>
            이러한 제품들을 사용해 보는 건 어떠세요?
          </h2>
          <div
            class="sc-ksluID kCniuP"
          >
            <a
              class="sc-hBMUJo hsXmSB"
              href="/detail/1"
            >
              <div
                class="sc-dIsUp fYttkb"
              >
                <img
                  alt="추천 상품 이미지"
                />
                <p>
                  Dummy Re...
                </p>
                <p>
                  2000
                   원
                </p>
              </div>
            </a>
          </div>
        </div>
        <div
          class="sc-bdnxRM deIdvJ"
        >
          <div
            class="sc-bdnxRM deIdvJ"
          >
            <div
              class="sc-gtsrHT bWIdmj"
            >
              <div
                class="sc-dlnjwi eDsHCx"
              >
                <img
                  alt="친환경마크"
                  class="sc-hKFxyN daLvAV"
                  src="https://image.edaily.co.kr/images/Photo/files/NP/S/2020/01/PS20012800634.jpg"
                />
                <div>
                  <p
                    class="certification-name"
                  >
                    저탄소 인증 제품
                  </p>
                  <p>
                    이 상품은 탄소가 적게 배출되는 상품으로 친환경적인 상품입니다.
                    <br />
                    저탄소제품은 환경성적표지 인증을 받은 제품 중 ‘저탄소 제품 기준’ 고시에 적합한 제품을 말합니다.
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="sc-eCApnc jPPoLs"
        >
          <p>
            Dummy Description
          </p>
          <img
            alt="제품 상세 이미지"
            src="https://cdn.pixabay.com/photo/2020/07/21/16/24/landscape-5426755_640.jpg"
          />
          <p>
            Dummy Description
          </p>
          <img
            alt="제품 상세 이미지"
            src="https://cdn.pixabay.com/photo/2020/07/21/16/24/landscape-5426755_640.jpg"
          />
          <p>
            Dummy Description
          </p>
          <img
            alt="제품 상세 이미지"
            src="https://cdn.pixabay.com/photo/2020/07/21/16/24/landscape-5426755_640.jpg"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="layout_container"
    >
      <div
        class="sc-crzoAE hXnyKs layout_container"
      >
        <img
          alt="상품 이미지"
          src="https://cdn.pixabay.com/photo/2020/07/21/16/24/landscape-5426755_640.jpg"
        />
        <div
          class="sc-jSFjdj iUSrcn"
        >
          <div
            class="sc-gKAaRy gQlVXN"
          >
            <h2
              class="title"
            >
              Dummy Name
            </h2>
            <h3
              class="price"
            >
              1000
               원
            </h3>
          </div>
          <div
            class="sc-iCoGMd cAGgJW"
          >
            <p>
              <span>
                삼성카드 간편결제 시 5% 청구할인
              </span>
               (최대 2천원)
            </p>
          </div>
          <div
            class="sc-fujyAs htYleG"
          >
            <p
              class="title"
            >
              탄소 노노 유저만을 위한 혜택
            </p>
            <p
              class="max-point"
            >
              최대 적립 포인트: 
              10
              원
            </p>
            <p
              class="basic-point"
            >
               ㄴ기본적립: 
              4
              원
            </p>
            <p
              class="extra-point"
            >
              맴버십 추가 적립: 
              6
              원
            </p>
          </div>
          <div
            class="sc-pNWdM ghyfJQ"
          >
            <p
              class="title"
            >
              추가상품
            </p>
            <div
              class="MuiFormControl-root makeStyles-formControl-1"
            >
              <label
                class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
                data-shrink="false"
                id="demo-simple-select-outlined-label"
              >
                추가상품 1
              </label>
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
              >
                <div
                  aria-haspopup="listbox"
                  aria-labelledby="demo-simple-select-outlined-label demo-simple-select-outlined"
                  class="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input"
                  id="demo-simple-select-outlined"
                  role="button"
                  tabindex="0"
                >
                  <span>
                    ​
                  </span>
                </div>
                <input
                  aria-hidden="true"
                  class="MuiSelect-nativeInput"
                  tabindex="-1"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7 10l5 5 5-5z"
                  />
                </svg>
                <fieldset
                  aria-hidden="true"
                  class="PrivateNotchedOutline-root-3 MuiOutlinedInput-notchedOutline"
                >
                  <legend
                    class="PrivateNotchedOutline-legendLabelled-5"
                  >
                    <span>
                      additional-product
                    </span>
                  </legend>
                </fieldset>
              </div>
            </div>
          </div>
          <div
            class="sc-kEqXSa lnzvgU"
          >
            <div
              class="sc-jrsJWt isZYCS"
            >
              <button
                aria-label="delete"
                class="MuiButtonBase-root MuiIconButton-root"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiIconButton-label"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 13H5v-2h14v2z"
                    />
                  </svg>
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
              <h4
                class="quantity"
              >
                1
              </h4>
              <button
                aria-label="delete"
                class="MuiButtonBase-root MuiIconButton-root"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiIconButton-label"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                    />
                  </svg>
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
            </div>
            <span
              class="total_quantity"
            >
              총 수량:  1개  |  
            </span>
            <h3
              class="total_price"
            >
              총 가격: 1000원
            </h3>
          </div>
          <div
            class="sc-iqAclL diZOqr"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeLarge"
              focusable="false"
              style="width: 100px; height: 100px; color: rgb(229, 229, 229);"
              viewBox="0 0 24 24"
            >
              <path
                d="M15.55 13c.75 0 1.41-.41 1.75-1.03l3.58-6.49c.37-.66-.11-1.48-.87-1.48H5.21l-.94-2H1v2h2l3.6 7.59-1.35 2.44C4.52 15.37 5.48 17 7 17h12v-2H7l1.1-2h7.45zM6.16 6h12.15l-2.76 5H8.53L6.16 6zM7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zm10 0c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2z"
              />
            </svg>
            <p>
              이 상품은 현재 구매하실 수 없는 상품입니다.
            </p>
            <p>
              (재입고 시 구매가능)
            </p>
          </div>
        </div>
      </div>
      <div
        class="sc-bqGGPW fiXlQZ"
      >
        <h2>
          이러한 제품들을 사용해 보는 건 어떠세요?
        </h2>
        <div
          class="sc-ksluID kCniuP"
        >
          <a
            class="sc-hBMUJo hsXmSB"
            href="/detail/1"
          >
            <div
              class="sc-dIsUp fYttkb"
            >
              <img
                alt="추천 상품 이미지"
              />
              <p>
                Dummy Re...
              </p>
              <p>
                2000
                 원
              </p>
            </div>
          </a>
        </div>
      </div>
      <div
        class="sc-bdnxRM deIdvJ"
      >
        <div
          class="sc-bdnxRM deIdvJ"
        >
          <div
            class="sc-gtsrHT bWIdmj"
          >
            <div
              class="sc-dlnjwi eDsHCx"
            >
              <img
                alt="친환경마크"
                class="sc-hKFxyN daLvAV"
                src="https://image.edaily.co.kr/images/Photo/files/NP/S/2020/01/PS20012800634.jpg"
              />
              <div>
                <p
                  class="certification-name"
                >
                  저탄소 인증 제품
                </p>
                <p>
                  이 상품은 탄소가 적게 배출되는 상품으로 친환경적인 상품입니다.
                  <br />
                  저탄소제품은 환경성적표지 인증을 받은 제품 중 ‘저탄소 제품 기준’ 고시에 적합한 제품을 말합니다.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="sc-eCApnc jPPoLs"
      >
        <p>
          Dummy Description
        </p>
        <img
          alt="제품 상세 이미지"
          src="https://cdn.pixabay.com/photo/2020/07/21/16/24/landscape-5426755_640.jpg"
        />
        <p>
          Dummy Description
        </p>
        <img
          alt="제품 상세 이미지"
          src="https://cdn.pixabay.com/photo/2020/07/21/16/24/landscape-5426755_640.jpg"
        />
        <p>
          Dummy Description
        </p>
        <img
          alt="제품 상세 이미지"
          src="https://cdn.pixabay.com/photo/2020/07/21/16/24/landscape-5426755_640.jpg"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
